common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -1671,7 +1671,8 @@ func (s *ExecutionManagerSuite) TestDeleteWorkflow() {
 // TestDeleteCurrentWorkflow test
 func (s *ExecutionManagerSuite) TestDeleteCurrentWorkflow() {
 	if s.ExecutionManager.GetName() != "cassandra" {
-		s.T().Skip("this test is only applicable for cassandra (uses TTL based deletes)")
+		return
+		//s.T().Skip("this test is only applicable for cassandra (uses TTL based deletes)")
```

Comment:
Skip here is wrong. this will skip more tests than it thought.

Reply:
maybe rephrase to comment to be just `"this test is only applicable for cassandra (uses TTL based deletes)"`